kernel/os/src/arch/cortex_m4/os_fault.c

```diff
@@ -76,7 +76,11 @@ struct coredump_regs {
     uint32_t psr;
 };
 
-#if MYNEWT_VAL(OS_COREDUMP)
+#if MYNEWT_VAL(OS_COREDUMP_CB)
+static coredump_cb_t g_coredump_cb = NULL;
```

Comment:
Remove the NULL assignment here, not needed.

Reply:
Removed